src/infinite-list/index.tsx

```diff
@@ -39,8 +41,10 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
     onReachNearEdgeThreshold,
     initialPageIndex = 0,
     extendedState,
-    scrollViewProps
+    scrollViewProps,
+    positionIndex = 0
```

Comment:
What's the purpose of `positionIndex`? 
Is that like `initialPageIndex`?

Reply:
It's like the index of the item where we need to append the data when updating with new items.

Reply:
The name is a little vague IMO, can you think of another name?

Reply:
`dataAppendIndex` is more descriptive I guess. Done

Reply:
Sorry. The change is here: https://github.com/wix/react-native-calendars/pull/1851